jvm-libs/linea/blob-decompressor/src/test/kotlin/net/consensys/linea/blob/BlobDecompressorDataDecodingTest.kt

```diff
@@ -0,0 +1,50 @@
+package net.consensys.linea.blob
+
+import net.consensys.linea.testing.filesystem.findPathTo
+import org.apache.tuweni.bytes.Bytes
+import org.assertj.core.api.Assertions.assertThat
+import org.hyperledger.besu.ethereum.core.Block
+import org.hyperledger.besu.ethereum.mainnet.MainnetBlockHeaderFunctions
+import org.hyperledger.besu.ethereum.rlp.RLP
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import kotlin.io.path.readBytes
+
+class BlobDecompressorDataDecodingTest {
+  private lateinit var decompressor: BlobDecompressor
+
+  @BeforeEach
+  fun beforeEach() {
+    decompressor = GoNativeBlobDecompressorFactory.getInstance(BlobDecompressorVersion.V1_1_0)
+  }
+
+  @Test
+  fun `can deserialize native lib testdata blobs`() {
+    val blob = findPathTo("prover")!!
+      .resolve("lib/compressor/blob/testdata/v0/sample-blob-0151eda71505187b5.bin")
+      .readBytes()
+    val decompressedBlob = decompressor.decompress(blob)
+    val blocksRlpEncoded = rlpDecodeAsListOfBytes(decompressedBlob)
+    assertThat(blocksRlpEncoded).hasSize(254)
```

Comment:
Here as well, why don't we check the data itself?

Reply:
See my comment [here](https://github.com/Consensys/linea-monorepo/pull/230#discussion_r1812261637)